### PR TITLE
[Feature ]: Display Voltage or GoodReserve logo when on Fuse or Eth network respectively

### PR DESCRIPTION
**Pull request by 1ucius**
# Description

Display Voltage or GoodReserve logo when on Fuse or Eth network respectively

About #112

# How Has This Been Tested?

Chrome Desktop

#UI:

Screenshot_12
Screenshot_13
Screenshot_14
Screenshot_15
Screenshot_16

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/pages/gd/Swap/index.tsx [hunk 1]
```diff
@@ -217,18 +222,28 @@ function Swap() {
...
+        }
+    ]
 
     if (!buying) pair.reverse()
 
+    const isFuse = SupportedChainId[Number(chainId)] === 'FUSE'
+
+    const swapHelperText = isFuse
+        ? i18n._(
+              t`FuseSwap is an UNI-V2 Automated Market Maker (AMM) that operates on Fuse Network where G$ is paired to another market tokens such as FUSE or USDC. The liquidity relies on Liquidity Providers that aggregate paired tokens to a pool. Price impact might be too hight when the swapping volume of one transaction is relatively hight to the total liquidity in the pool.`
```
Comment (L03TJ3) [inline src/pages/gd/Swap/index.tsx:241]: Shouldn't this read Voltage instead of fuseSwap?

File: src/components/Icon/index.tsx [hunk 2]
```diff
@@ -55,3 +55,49 @@ export const TwitterIcon = createIcon(
...
...PpTt+IbawOI8j+PrSCZdtrDJL7xPNaukKIA7rU2ZI+mtmNRf4geZQGdxS6iBUZgNwpouOAiFafzIbVG0Qqnj0h53cH77kvsVzTXUHtzpzVoIqPoMIuUa3Mrnq1nWUihSmiH8pvYxhpwkD5AOQWIwkBW+Pso19km5G/kVmfB0jc5C+UJ2Zz/gCmdQQK4jEy1OUKuhxdbBhT/DQJn5wOO1rRa1xEFBO/eyoWER9qIHhrZh+4cIR/mXv746aJv7sRKMd3I01e6B9HkSFSm59/BIHwPC+iVJt6sYVjg0nq7ikcB4zNcZ8eKJLTUrtL3aPsOYiFdUHSDTjuq+3dRL2ewVSTHNJ+QToOApGdfUHloIkXQmsZ4l/oDCekQJ7VnQ55oJX2m+cZIM6A5LKL3sUE6mwV0X/HZSvMy828F8iGdzwbpQ2yAtm/FszRGgLMBU1mIcwG1C3flvjAuibxoafTbwLVdO71jSwQrKsJK5QJdQGqEDSmMJoAT6xqXZRbmD4dzQjUDdBqxxhBwNlSnt04jp7522TTHe5vLPXmk4/mXFN2n0+gOKRrU88A9/5uOkL6OecKa0ZK5XMgbDP3Q+JRaSzN+ptdc7R0wDmfTloo0E5H4IX2VDVLAAvodi1SyMzgtYMQCA1b6AFgeYaHKeon4lXNYQF9UQYcX03ap4jlqJWrtEbQPkf0DKmPlwWxQBSwiYBQxJJi6grv5oyxUAbV3sICuYQGtZCvo+KKh6xkzQx3XcrzRY2Ffc5h9lgV0RjG9NYDkL0XZMfr6GwtUsOeVxf74CgvpPewX9KZx7Y9zabpyEtGJ075VnO68tgjkCGKG7v9e7AuszWAk832GJHfDtKsV80Gc3uocK04znVicn8ccfbOfex2Rv9vx5FVcFIa5K0+BW3BXTu63rdu/Y6sdAuo0y/ow5JnZK3cG+WZG+F/Kuv55mBbjrv+njJtHfpjHuKuoXpA+AfFFl2dceRZ38//DvSX9XZ4/uxXAOctbvB0XhQEu/DX1Po7bfxqAMqlxWdG/Gxfy0npl4DhSYKdZX6pf37JpcD3vEvm+rJs/XL1MqLrlH5BuwvHkI9wtgJxV6U49E668kHvyeMf13z27R+7c3b1sk/Lys95eGbjEJ8mfdf1gzvzCjPL78OLkqDTZSfqTv8q6b5tffl+9/f8HVY4EozWeUCMAAAAASUVORK5CYII="
```
Comment (L03TJ3) [inline src/components/Icon/index.tsx:74]: isn't it better to add the icon to assets/icon
and then (ie): 
`import GoodReserveLogo from 'assets/icon/goodreserve.png'`

`xlinkHref = GoodReserveLogo`

Same for the voltage one

**Comment by 1ucius** [comment]
@L03TJ3 please, review

**Review by L03TJ3 — APPROVED** [review]

**Comment by sirpy** [comment]
@L03TJ3 @1ucius the info text has spelling mistakes and it says fuseswap instead of voltage swap

**Comment by L03TJ3** [comment]
@sirpy should it read voltage swap instead or just Voltage? that's what I requested

I'll update the other spelling mistakes along with a pending styling update I have now

**Pull request merged**